model/textparse/openmetricsparse.go

```diff
@@ -94,16 +94,51 @@ type OpenMetricsParser struct {
 	exemplarVal   float64
 	exemplarTs    int64
 	hasExemplarTs bool
+
+	// lines ending with _created are skipped by default as they are
+	// parsed by the CreatedTimestamp method. The skipCT flag is set to
+	// false when the CreatedTimestamp method is called.
+	skipCT bool
+}
+type openMetricsParserOptions struct {
+	// SkipCT skips the parsing of _created lines by default.
+	SkipCT bool
+}
+
+type OpenMetricsOption func(*openMetricsParserOptions)
+
+func WithSkipCT(skipCT bool) OpenMetricsOption {
+	return func(o *openMetricsParserOptions) {
+		o.SkipCT = skipCT
+	}
 }
 
 // NewOpenMetricsParser returns a new parser of the byte slice.
 func NewOpenMetricsParser(b []byte, st *labels.SymbolTable) Parser {
 	return &OpenMetricsParser{
 		l:       &openMetricsLexer{b: b},
 		builder: labels.NewScratchBuilderWithSymbolTable(st, 16),
+		skipCT:  true,
 	}
 }
 
+// NewOpenMetricsParserWithOpts returns a new parser of the byte slice with options.
+func NewOpenMetricsParserWithOpts(b []byte, st *labels.SymbolTable, opts ...OpenMetricsOption) Parser {
+	parser := &OpenMetricsParser{
+		l:       &openMetricsLexer{b: b},
+		builder: labels.NewScratchBuilderWithSymbolTable(st, 16),
+	}
+	OpenMetricsParserOptions := openMetricsParserOptions{
```

Comment:
Maybe we could make the intention of this variable clearer?
```suggestion
	DefaultOpenMetricsParserOptions := openMetricsParserOptions{
```

What do you think?

Reply:
Sounds good!